include/aikido/planner/vectorfield/VectorFieldUtil.hpp

```diff
@@ -67,22 +68,67 @@ class DesiredTwistFunction : public dart::optimizer::Function
 /// Compute joint velocity from a given twist.
 ///
 /// \param[in] _desiredTwist Desired twist, which consists of angular velocity
-/// and linear velocity
+/// and linear velocity.
 /// \param[in] _stateSpace MetaSkeleton state space
 /// \param[in]  _bodyNode Body node of the end-effector
+/// \param[in] _jointLimitTolerance If less then this distance to joint
+/// limit, velocity is bounded in that direction to 0.
+/// \param[in] _jointVelocityLowerLimits Joint velocity lower bounds
+/// \param[in] _jointVelocityUpperLimits Joint velocity upper bounds
+/// \param[in] _jointVelocityLimited Whether joint velocity bounds are
+/// considered in optimization
+/// \param[in] _maxStepSize Max step size in second.
 /// \param[in] _optimizationTolerance Callback of vector field calculation
-/// \param[in] _timestep How long will the computed joint velocities be executed
-/// \param[in] _padding Padding for joint limits
 /// \param[out] _jointVelocity Calculated joint velocities
 bool computeJointVelocityFromTwist(
     const Eigen::Vector6d& _desiredTwist,
     const aikido::statespace::dart::MetaSkeletonStateSpacePtr _stateSpace,
     const dart::dynamics::BodyNodePtr _bodyNode,
+    double _jointLimitTolerance,
+    const Eigen::VectorXd& _jointVelocityLowerLimits,
+    const Eigen::VectorXd& _jointVelocityUpperLimits,
```

Comment:
The original reason is allowing set joint velocity limits as `INF` like `prpy`. 
But this will cause `nlopt` error. I thus added `_jointVelocityLimited` . 
I still keep these two arguments, assuming users might set joint velocity limits that are different with meta skeleton's. (for example, tighter bounds or looser bounds). What do you think?